aws/cloudformation/bootstrap_chef_stack.sh.erb

```diff
@@ -113,7 +113,7 @@ cat <<JSON > $FIRST_BOOT
   },
   "cdo-secrets": {
 <% if frontends -%>
-    "session_store_server": "redis://${SessionStoreGroup.PrimaryEndPoint.Address}:${SessionStoreGroup.PrimaryEndPoint.Port}",
+    "session_store_server": "rediss://${SessionStoreGroup.PrimaryEndPoint.Address}:${SessionStoreGroup.PrimaryEndPoint.Port}",
```

Comment:
Note that [because we have enabled encryption for the underlying Redis cluster](https://github.com/code-dot-org/code-dot-org/blob/cf805a0e720f0c35f4d0a8b00c3633a59f71b3c3/aws/cloudformation/components/session_store.yml.erb#L21), we have to specifically use a "redis secure" connection here, hence `rediss` rather than `redis`.

I very briefly tried to find a way to automatically determine from whether or not that was necessary based on the CloudFormation config, but couldn't so decided to just hardcode it. Let me know if you think this is worth spending some time on!